projects/python/project_ideas.md

```diff
@@ -0,0 +1,35 @@
+Python
```

Comment:
only need one idea, so pick your "best" for now

Reply:
I picked one of the python projects for my project and attached the link.